From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kevin Venalainen <kevin.venalainen@around.co>
Date: Fri, 1 Oct 2021 11:12:03 -0700
Subject: removes special case handling for airpods name

This patch removes the special case handling for all AirPods' names.

diff --git a/third_party/blink/renderer/modules/mediastream/media_devices.cc b/third_party/blink/renderer/modules/mediastream/media_devices.cc
index 4b4377251af7c09aaa836a2ee9dc991cea386bb3..22069b56b8e41045ca8199346c801eb547aa89e8 100644
--- a/third_party/blink/renderer/modules/mediastream/media_devices.cc
+++ b/third_party/blink/renderer/modules/mediastream/media_devices.cc
@@ -463,9 +463,12 @@ void MediaDevices::DevicesEnumerated(
           static_cast<mojom::blink::MediaDeviceType>(i);
       WebMediaDeviceInfo device_info = enumeration[i][j];
       String device_label = String::FromUTF8(device_info.label);
-      if (device_label.Contains("AirPods")) {
-        device_label = "AirPods";
-      }
+      // Chromium replaces this because the device name can contain a user's
+      // name which they hide for privacy reasons. Around desktopapp doesn't
+      // need to do this. Changing this label also messes with audio sharing.
+      // if (device_label.Contains("AirPods")) {
+      //   device_label = "AirPods";
+      // }
       if (device_type == mojom::blink::MediaDeviceType::MEDIA_AUDIO_INPUT ||
           device_type == mojom::blink::MediaDeviceType::MEDIA_VIDEO_INPUT) {
         InputDeviceInfo* input_device_info =
